instrumentation/log4j/log4j-appender-2.17/library/src/main/java/io/opentelemetry/instrumentation/log4j/appender/v2_17/OpenTelemetryAppender.java

```diff
@@ -44,8 +42,8 @@ public class OpenTelemetryAppender extends AbstractAppender {
 
   static final String PLUGIN_NAME = "OpenTelemetry";
 
-  private static final LogEmitterProviderHolder logEmitterProviderHolder =
-      new LogEmitterProviderHolder();
+  private static final AtomicReference<LoggerProvider> loggerProviderRef =
+      new AtomicReference<>(LoggerProvider.noop());
```

Comment:
Should we just use the `GlobalLoggerProvider` instead of these?
Previously, each of these appenders had to have a separate global instance of the logger provider cause haven't had one in the appender-api, but now that we've added it to the logs-api we could use it (and we tend to use the `GlobalOpenTelemetry` in the few library instrumentations that cannot be configured "normally", e.g. JDBC)

Reply:
Yes, I think so. Meant to ask you all about what you thought about this 🙂

Reply:
Right... alternatively, we could add a weak map/`VirtualField` that'd map from application logger to OTel logger.

Reply:
I'm thinking we just call `GlobalOpenTelemetry` for now, since its no different that the previous behavior from a performance standpoint. But in the future I'm going to try to do some optimization on the hot path in the log sdk, and maybe then we can revisit whether we can do better than `GlobalOpenTelemetry.get()`.